src/components/PageTitle.tsx

```diff
@@ -1,57 +1,111 @@
 import React from "react";
 import { Skeleton } from "antd";
-import { Subtitle } from "@leafygreen-ui/typography";
+import { H2, Subtitle } from "@leafygreen-ui/typography";
 import styled from "@emotion/styled";
+import { css } from "@emotion/core";
+
+type Size = "large" | "medium";
 
 interface Props {
   loading: boolean;
   hasData: boolean;
   title: string;
   badge: JSX.Element;
   buttons?: JSX.Element;
+  size?: Size;
+}
+
+interface TitleTypographyProps {
+  size: Size;
 }
 
+const TitleTypography: React.FC<TitleTypographyProps> = ({
+  children,
+  size,
+}) => {
+  if (size === "large") {
+    return <H2>{children}</H2>;
+  }
+  if (size === "medium") {
+    return <Subtitle>{children}</Subtitle>;
+  }
+};
+
 export const PageTitle: React.FC<Props> = ({
   loading,
   hasData,
   title,
   badge,
   buttons,
+  size = "medium",
 }) => (
   <>
     {!hasData && loading && (
-      <PageHeader>
+      <PageHeader size={size}>
         <Skeleton active paragraph={{ rows: 0 }} />
       </PageHeader>
     )}
     {hasData && !loading && (
-      <PageHeader>
-        <TitleWrapper>
-          <Subtitle>
+      <PageHeader size={size}>
+        <TitleWrapper size={size}>
+          <TitleTypography size={size}>
             <span data-cy="page-title">
               {title}
               {"  "}
-              <BadgeWrapper>{badge}</BadgeWrapper>
+              <BadgeWrapper size={size}>{badge}</BadgeWrapper>
             </span>
-          </Subtitle>
+          </TitleTypography>
         </TitleWrapper>
         {buttons ?? null}
       </PageHeader>
     )}
   </>
 );
-const BadgeWrapper = styled.span`
-  display: inline-flex;
-  position: relative;
-  top: -2px;
+const BadgeWrapper = styled.span<TitleTypographyProps>`
+  ${({ size }) =>
+    size === "medium" &&
+    css`
+      display: inline-flex;
+      position: relative;
+      top: -2px;
```

Comment:
instead of having two separate string literals, this can be done: 

`top: ${props => props.size === "medium" ? "-2px" : "-4px"}`;